tools/generate-helpers/pg-table-bindings/testgraphtables/testg3grandchild1/store.go

```diff
@@ -202,10 +202,13 @@ func (s *storeImpl) upsert(ctx context.Context, objs ...*storage.TestG3GrandChil
 			_, err := batchResults.Exec()
 			result = multierror.Append(result, err)
 		}
-		batchResults.Close()
+		closeErr := batchResults.Close()
 		if err := result.ErrorOrNil(); err != nil {
 			return err
 		}
+		if closeErr != nil {
```

Comment:
If would just do if err := batchResults.Close(); err != nil

I don't think we should try to look at result